brianlong/www.validators.app
# 181957663 update chartjs

#### What's this PR do?
- Updates all chart on the page to 3.9.1 syntax
- Get rid of deprecated gauge chart and write custom plugin for speedometer 
- Move validator charts to correct directory

#### How should this be manually tested?
- Test on staging to display charts with as accurate data as possible
- Go through all rails&vue.js pages and check them carrefully. Please compare each chart against production (the list of charts in PT story - you can tick verified charts on the list). Confirm that there're no significant differences and data is displayed correctly.
- Send some ping things to staging (https://stage.validators.app/api-documentation?locale=en&network=mainnet#ping-thing) and confirm that ping thing charts are still updated live, same as on production (https://stage.validators.app/ping-thing?locale=en&network=mainnet)

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/181957663)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
